### PR TITLE
Add quickstart guide

### LINKED ISSUE #44: Create a quickstart install guide
This issue is related to creating a quickstart install guide using the automation.

- Divide the variables into multiple sections.
- Mark variables as Required or Optional.
- Improve the documentation.

### PR DESCRIPTION
**Pull request by yussufsh**
Fixes #44

Signed-off-by: Yussuf Shaikh <yussuf@us.ibm.com>

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+```
+git clone https://github.com/ocp-power-automation/ocp4-upi-powervm.git
+cd ocp4_upi_powervm
+```
+
+## Setup Variables.
+
+Update the var.tfvars with values explained in the following sections. You can also set the variables using other ways mentioned [here](https://www.terraform.io/docs/configuration/variables.html#assigning-values-to-root-module-variables) such as -var option or environment variables.
+
+### Setup PowerVC Environment Variables
```
Comment (bpradipt) [inline docs/quickstart.md:24]: This way of organizing the variables makes it much easier to understand. Thanks.
Is it possible to group the `Required` and `Optional` together ?

Comment (yussufsh) [reply docs/quickstart.md:24]: Do we want a new sub-section on top under variables with only all the required variables?

Comment (yussufsh) [reply docs/quickstart.md:24]: Or we just keep a generic note that you need to fill in all the variables marked as (Required) below?

Comment (bpradipt) [reply docs/quickstart.md:24]: no subsection.. just changing the order.
From this
```
var1 (required)
var2 (required)
var3 (optional)
var4 (required)
var5 (optional)
```
to the following
```
var1 (required)
var2 (required)
var4 (required)

var3 (optional)
var5 (optional)
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+ * `keypair_name` : (Optional) Value for keypair used. Default is <cluster_id>-keypair.
+ * `public_key_file` : (Optional) A pregenerated OpenSSH-formatted public key file. Default path is 'data/id_rsa.pub'.
+ * `private_key_file` : (Optional) Corresponding private key file. Default path is 'data/id_rsa'.
+ * `private_key` : (Optional) The contents of an SSH key to use for the connection. Ignored if `public_key_file` is provided.
+ * `public_key` : (Optional) The contents of corresponding key to use for the connection. Ignored if `public_key_file` is provided.
+ * `rhcos_kernel_options` : (Optional) List of [kernel arguments](https://docs.openshift.com/container-platform/4.4/nodes/nodes/nodes-nodes-working.html#nodes-nodes-kernel-arguments_nodes-nodes-working) for the cluster nodes eg: ["slub_max_order=0","loglevel=7"]. Note that this will be applied after the cluster is installed, hence wait till all the nodes are in `Ready` status before you start using the cluster. Check nodes status using the command `oc get nodes`.
+
+### Setup Openshift Variables
```
Comment (bpradipt) [inline docs/quickstart.md]: OpenShift

File: docs/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+ * `storage_type` : (Optional) Storage provisioner to configure. Supported values: nfs (For now only nfs provisioner is supported, any other value won't setup a storageclass)
+ * `volume_size` : (Optional) If storage_type is nfs, a volume will be created with given size (default 300) in GB and attached to bastion node. Eg: 1000 for 1TB disk.
+ * `volume_storage_template` : (Optional) Storage template name or ID for creating the volume. Empty value will use default template.
+
+### Setup OCP Upgrade Variables
+
+Update the following variables specific to OCP upgrade. The upgrade will be performed after a successful install of OCP.
+
+ * `upgrade_image` : (Optional) Openshift release image having higher and supported version. If set, OCP cluster will be upgraded to this image version. (e.g. `"quay.io/openshift-release-dev/ocp-release-nightly@sha256:552ed19a988fff336712a50..."`)
```
Comment (bpradipt) [inline docs/quickstart.md]: OpenShift

**Review by bpradipt — APPROVED** [review]
lgtm

**Review by Prajyot-Parab — APPROVED** [review]
LGTM!

**Pull request merged**